src/nanaimo/version.py

```diff
@@ -3,6 +3,6 @@
 # This software is distributed under the terms of the MIT License.
 #
 
-__version__ = 0, 0, 5
+__version__ = 0, 0, 6
```

Comment:
This should be a string per [PEP-396](https://www.python.org/dev/peps/pep-0396/#specification).

This used to be done incorrectly in PyUAVCAN and PyDSDL but I fixed that some time ago.

Some packages out there use [`__version_info__`](https://stackoverflow.com/questions/458550/standard-way-to-embed-version-into-python-package) for a tuple of version numbers, but as far as I know this convention is not explicitly documented anywhere.

Reply:
https://github.com/search?l=Python&q=__version_info__&type=Code

Reply:
Thanks! I'll just change it to a string. That's actually easier to deal with ultimately.